ui/pages/send/send-content/send-asset-row/send-asset-row.container.js

```diff
@@ -1,20 +1,25 @@
 import { connect } from 'react-redux';
-import { getNativeCurrency } from '../../../../ducks/metamask/metamask';
+import {
+  getCollectibles,
+  getNativeCurrency,
+} from '../../../../ducks/metamask/metamask';
 import {
   getMetaMaskAccounts,
   getNativeCurrencyImage,
 } from '../../../../selectors';
-import { updateSendAsset, getSendAssetAddress } from '../../../../ducks/send';
+import { updateSendAsset, getSendAsset } from '../../../../ducks/send';
 import SendAssetRow from './send-asset-row.component';
 
 function mapStateToProps(state) {
   return {
     tokens: state.metamask.tokens,
     selectedAddress: state.metamask.selectedAddress,
-    sendAssetAddress: getSendAssetAddress(state),
+    collectibles: getCollectibles(state),
+    sendAsset: getSendAsset(state),
     accounts: getMetaMaskAccounts(state),
     nativeCurrency: getNativeCurrency(state),
     nativeCurrencyImage: getNativeCurrencyImage(state),
+    testState: state,
```

Comment:
Is this supposed to be here?

Reply:
nope 😶‍🌫️

Reply:
removed here https://github.com/MetaMask/metamask-extension/pull/13048/commits/fea5a50556620b3be66fee55361447c033ea5de2